packages/react/src/themes/font-awesome/components/Icon/iconStyles.ts

```diff
@@ -3,7 +3,7 @@ import { IconProps } from '../../../../components/Icon/Icon'
 import { IconVariables } from '../../../teams/components/Icon/iconVariables'
 
 const iconStyles: ComponentSlotStylesInput<IconProps, IconVariables> = {
-  fontRoot: (): ICSSInJSStyle => ({
+  root: (): ICSSInJSStyle => ({
```

Comment:
Shuold we add this conditionally if the icon is font icon? Won't this break icons on docs?

Reply:
I check that it doesn't break anything, so not sure if you should implement the comment...

Reply:
agree that it is better to introduce explicit condition check here, as we are expecting this rule to be applied exclusively to font-based icons